staging/src/k8s.io/apiserver/pkg/server/options/serving.go

```diff
@@ -321,7 +322,8 @@ func CreateListener(network, addr string) (net.Listener, int, error) {
 	if len(network) == 0 {
 		network = "tcp"
 	}
-	ln, err := net.Listen(network, addr)
+	config := &net.ListenConfig{Control: reusePort}
+	ln, err := config.Listen(context.Background(), network, addr)
```

Comment:
I think I'd prefer `context.TODO()` here-- we might want to pass this in.

Reply:
Sure, done.